### PR TITLE
[REG-1404] Fix Windows Sample Import

### PR DESCRIPTION
**Pull request by rcornwall-reg**
**Changes**
* Updated Docs URL
* Fixed paths to work on both Windows and Mac

**To Test**
1. Create new URP Project on Mac
2. Import SDK
3. Hit "Continue as Guest"
4. Hit "Import Sample" and verify it opens
5. Create new URP Project on PC
6. Import SDK
7. Hit "Continue as Guest"
8. Hit "Import Sample" and verify it opens

### CODE REVIEW

File: Editor/Scripts/Startup/RegressionPackagePopup.cs
Comment (RG-nAmKcAz) [inline Editor/Scripts/Startup/RegressionPackagePopup.cs:370]: I think we should stick with using `Path.Combine` instead of assumed `/` pathing.

the reason it wasn't working before is that the first argument must be an absolute path.

something like this

```csharp
string samplePath = Path.Combine(FileUtil.GetPhysicalPath("Samples~"), "ThirdPersonDemoURP");
string destinationPath = Path.Combine(FileUtil.GetPhysicalPath("Assets"), "ThirdPersonDemoURP");
string packagePath = Path.Combine(FileUtil.GetPhysicalPath("Packages"), packageName, samplePath);

//...


EditorSceneManager.OpenScene(Path.Combine(destinationPath, "Demo", "Scenes", "Playground.unity"));
```

**Review by RG-nAmKcAz — CHANGES_REQUESTED** [review]

**Review by vontell — APPROVED** [review]
Approving assuming that Zack's changes are approved. Approving so you aren't blocked once he approves.

**Review by abeizer — APPROVED** [review]
Approving pending Zack's comments

**Comment by rcornwall-reg** [comment]
Ready for re review ✅ 
Tested on Mac and Windows ✅ 

* Used `Path.Combine(...).Replace("\\", "/");` to maintain requirement of forward slashes in directory paths for [Unity's File Utils](https://docs.unity3d.com/ScriptReference/FileUtil.CopyFileOrDirectory.html)
* `EditorSceneManager` uses full absolute name from `Application.dataPath`

**Review by RG-nAmKcAz — APPROVED** [review]

**Pull request merged**
